python/lsst/tmpl/__init__.py

```diff
@@ -1,28 +1,26 @@
-# 
-# LSST Data Management System
 #
-# Copyright 2008-2017  AURA/LSST.
-# 
-# This product includes software developed by the
-# LSST Project (http://www.lsst.org/).
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
 #
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation, either version 3 of the License, or
 # (at your option) any later version.
-# 
+#
 # This program is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
-# 
-# You should have received a copy of the LSST License Statement and 
-# the GNU General Public License along with this program.  If not, 
-# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 
 from .version import *   # generated by sconsUtils unless you tell it not to
-from .exampleOne import *
-from .exampleTwo import *
-from .exampleThree import *
 
+from ._tmpl import *
+from ._ExampleTwo import *
```

Comment:
Perhaps make clear here with a comment that this is needed because of the Python extension, not the wrappers (which are included in `_tmpl`)?